brain-marks/Services/MigrationService.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  MigrationService.swift
+//  brain-marks
+//
+//  Created by Jay on 12/13/22.
+//
+
+import CoreData
+import Foundation
+
+/// Controls the migration from Amplify to CoreData
+class MigrationService {
```

Comment:
```suggestion
final class MigrationService {
```

Because this doesn't need to be inherited by anything

Reply:
Done ✅